### PR TITLE
Proj 9/Created list of tasks

### PR DESCRIPTION
**Pull request by kenzywa**
Created list of tasks with properties of task.

### CODE REVIEW

File: TableApplication/Task/Task.swift [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+struct Task {
+    var title : String
+    var id : String
+    var description : String
+    var start_date : String
```
Comment (damencki) [inline TableApplication/Task/Task.swift]: should be date

File: TableApplication/Task/Task.swift [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    var finish_date : String
+    var status : String
```
Comment (damencki) [inline TableApplication/Task/Task.swift]: should be enum

File: TableApplication/Task/Task.swift [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    var employee : String
```
Comment (damencki) [inline TableApplication/Task/Task.swift]: incorrect type

File: TableApplication/Task/Task.swift [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    var project : String
```
Comment (damencki) [inline TableApplication/Task/Task.swift]: incorrect type

File: TableApplication/Task/TaskListViewController.swift [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+        createTable()
+        setupUI()
+    }
+    //MARK: - Methods
+    func createTable() {
+        self.taskListTableView.frame = view.bounds
+        taskListTableView.register(UITableViewCell.self, forCellReuseIdentifier: identifier)
+        self.taskListTableView.delegate = self
+        self.taskListTableView.dataSource = self
+        taskListTableView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
```
Comment (damencki) [inline TableApplication/Task/TaskListViewController.swift]: use autolayout or autoresizing mask

File: TableApplication/Task/TaskListViewController.swift [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: identifier, for: indexPath)
+        let task = tasks[indexPath.row]
+        cell.textLabel?.text = "\(task.title)"
+        cell.textLabel?.font = UIFont(name: "San Francisco", size: 40.0)
+        return cell
+    }
+    
+//    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
```
Comment (damencki) [inline TableApplication/Task/TaskListViewController.swift]: delete comments

**Review by damencki — CHANGES_REQUESTED** [review]

File: TableApplication/Task/TaskListViewController.swift (hunk 6 above)
Comment (kenzywa) [reply TableApplication/Task/TaskListViewController.swift]: Solved

File: TableApplication/Task/Task.swift (hunk 4 above)
Comment (kenzywa) [reply TableApplication/Task/Task.swift]: Solved

File: TableApplication/Task/Task.swift (hunk 3 above)
Comment (kenzywa) [reply TableApplication/Task/Task.swift]: Solved

File: TableApplication/Task/Task.swift (hunk 2 above)
Comment (kenzywa) [reply TableApplication/Task/Task.swift]: Solved

File: TableApplication/Task/Task.swift (hunk 1 above)
Comment (kenzywa) [reply TableApplication/Task/Task.swift]: Solved

File: TableApplication/Task/TaskListViewController.swift (hunk 5 above)
Comment (kenzywa) [reply TableApplication/Task/TaskListViewController.swift]: Solved

File: TableApplication/Task/TaskListViewController.swift [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.title = Сonstants.titleOfView
+        createTable()
+        setupUI()
+    }
+    //MARK: - Methods
+    func createTable() {
+        self.taskListTableView.frame = view.bounds
```
Comment (damencki) [inline TableApplication/Task/TaskListViewController.swift]: delete setup frame

**Review by damencki — CHANGES_REQUESTED** [review]

**Review by damencki — APPROVED** [review]

**Pull request merged**